doc-app/tests/dummy/app/templates/docs/ember-input-validation/prefabs/phone.md

```diff
@@ -0,0 +1,15 @@
+# Input phone
+
+This is a phone input field.
```

Comment:
Je trouve que ça mérite de rajouter des petits détails genre le nombre de pays couvert mais as you want :p

Reply:
copieur